### DISCUSSION TITLE
What is the difference between Tooltip and HoverCard?

### DISCUSSION THREAD

**Discussion by piszczu4** [shadcn-ui/ui#2417] [Q&A]
Why should I use hover card when there is a Tooltip component as well? What is the difference between them?

**Comment by SilasRodrigues19** [comment]
Hey @piszczu4 

The `Tooltip` component is basically for something simpler, like literally some tip when you hover over it, while the `Hover Card` component already has a Card structure when you hover over it, so you can create something more detailed.

I looked at the API Reference and it seems that `Hover Card` is ignored by screen readers but you can check the differences and characteristics in the links below:

[Tooltip - API Reference](https://www.radix-ui.com/primitives/docs/components/tooltip#api-reference)

[Hover Card - API Reference](https://www.radix-ui.com/primitives/docs/components/hover-card)

**Comment by doooooooooots** [comment]
### Hover Card
[Accessibility](https://www.radix-ui.com/primitives/docs/components/hover-card#accessibility)
The hover card is intended for sighted users only, the content will be inaccessible to keyboard users.

---
### Tooltip
A popup that displays information related to an element when the element receives keyboard focus or the mouse hovers over it.
